src/components/datetimepicker/Datetimepicker.vue

```diff
@@ -166,8 +166,11 @@ export default {
             get() {
                 return this.newValue
             },
+
             set(value) {
                 if (value) {
+                    // check invalid date
+                    value = isNaN(value.getTime()) ? this.newValue : value
```

Comment:
This would only solves the issue on DateTimePicker, but I guess the problem exists with every components depending on TimePicker. The `TimepickerMixin.js` should be improve instead.

Reply:
Ok. I will solve it in the root of problem.

Reply:
I have tested the granularity on TimePicker and it works fine. Apparently the problem only occurs in the Datetimepicker. I noticed that when I click to select a date without time the problem is occurring. 

Here is after the component opened
![print](https://user-images.githubusercontent.com/15680379/133779723-5c21d486-3475-4e16-9d14-7a6e71fdebbb.png)

Here is after select a date (in this point the `input` is being emitted with `Invalid Date`).
![print2](https://user-images.githubusercontent.com/15680379/133779726-2a66d0ae-2fc5-473d-b7c8-69b39ea1e428.png)

Reply:
Note the timepicker. It was cleaned, or something, and after that I can't select time anymore.

Reply:
I'll debug to get more details about what is happening.